quickjs.c

```diff
@@ -50003,10 +49965,8 @@ static JSValue js_typed_array_indexOf(JSContext *ctx, JSValue this_val,
     } else
     if (tag == JS_TAG_FLOAT64) {
         d = JS_VALUE_GET_FLOAT64(argv[0]);
-        if (d >= INT64_MIN && d < 0x1p63) {
-            v64 = d;
-            is_int = (v64 == d);
-        }
+        v64 = d;
```

Comment:
is this safe to do unchecked?

Reply:
Unintended change, will revert. (possibly caused as a side effect of `git rebase`)

Reply:
👍